shared/gh/partials/admin-batch-edit-date.html

```diff
@@ -1,9 +1,83 @@
-<%
-    var startDate = utilAPI.convertUnixDatetoISODate(utilAPI.fixDateToGMT(data.start));
-    var endDate = utilAPI.convertUnixDatetoISODate(utilAPI.fixDateToGMT(data.end));
-%>
-<span class="gh-event-calendar-icon">
-    <span class="month"><%- utilAPI.dateDisplay(data.start).monthName() %></span>
-    <span class="day"><%- utilAPI.dateDisplay(data.start).dayNumber() %></span>
-</span>
-<span class="gh-event-display-date"><%- utilAPI.generateDisplayDate(startDate, endDate) %></span>
+<% var termNames = _.pluck(termsInUse, 'name'); %>
+<% var termLabels = _.pluck(termsInUse, 'label'); %>
+<div id="gh-batch-edit-date-term-description">
+    <small>Selected</small>
+    <h2>
+        <%- termLabels.join(', ') %>
+    </h2>
+</div>
+<div id="gh-batch-edit-date-picker-container" data-terms="<%- termNames.join(',') %>">
+    <!-- Show the number of weeks that are selected --> 
+    <small><%- weeksInUse.length %> week<% if (weeksInUse.length !== 1) { %>s<% } %> selected</small>
+    <div id="gh-batch-edit-date-picker">
+        <% var totalWeeks = numberOfWeeks%>
+        <!-- Keep rendering new weeks until the amount of weeks left to render is 0 -->
+        <% while (numberOfWeeks) { %>
+            <% numberOfWeeks--; %>
+            <% var weekNumber = totalWeeks - numberOfWeeks%>
+            <!-- Determine if the week should be checked or not -->
+            <% var weekUsed = _.indexOf(weeksInUse, weekNumber) !== -1 %>
+            <div class="checkbox <% if (weekUsed) { %>gh-batch-edit-date-picker-selected<% } %>">
+                <label>
+                    <input type="checkbox" value="<%- weekNumber %>" <% if (weekUsed) { %>checked<% } %>><%- gh.api.utilAPI.addLeadingZero(weekNumber) %> <i class="fa fa-square-o"></i><i class="fa fa-check-square-o"></i>
```

Comment:
It does but it doesn't actually give me any 'unchecked' states so I decided to go for the square